### PR TITLE
Print Alert string in pool list detail view

### LINKED ISSUE #903: Don't omit to show Alerts, if present, the pool list detail view
Right now, the only thing shown is pool availability actions. We should instead show the Alerts prominently, with all relevant alert codes displayed.

### PR DESCRIPTION
**Pull request by bgurney-rh**
Closes #903

### CODE REVIEW

**Comment by bgurney-rh** [comment]
Example output (with an intentional WS001 alert):
```
$ stratis pool list --uuid=050a7855-ac20-4289-b1eb-19dc5f267f6a
UUID: 050a7855-ac20-4289-b1eb-19dc5f267f6a
Name: spool1
Alerts: WS001
Actions Allowed: fully_operational
Cache: No
...
```

**Review by mulkieran — COMMENTED** [review]
Looks good so far. The question now is, should we go with your proposal from https://github.com/stratis-storage/stratis-cli/issues/903#issuecomment-1217167634, or should we stay with this? @drckeefe might have an opinion also.

**Comment by bgurney-rh** [comment]
I was sketching an idea of the PoolMaintenanceErrorCode errors, and PoolAllocSpaceErrorCode errors (from src/stratis_cli/_error_codes.py), to see if I could think of a shorter string to explain the current error codes:
```
WS001     NO_ALLOC_SPACE    All devices fully allocated
                "Every device belonging to the pool has been fully allocated. "
                "To increase the allocable space, add additional data devices "
                "to the pool."


EM001     NO_IPC_REQUESTS   Pool state changes not possible
                "The pool will return an error on any IPC request that could "
                "cause a change in the pool state, for example, a request to "
                "rename a filesystem. It will still be able to respond to "
                "purely informational requests."


EM002     NO_POOL_CHANGES   Pool maintenance operations not possible
                "The pool is unable to manage itself by reacting to events, "
                "such as devicemapper events, that might require it to take "
                "any maintenance operations."
```

**Comment by mulkieran** [comment]
> I was sketching an idea of the PoolMaintenanceErrorCode errors, and PoolAllocSpaceErrorCode errors (from src/stratis_cli/_error_codes.py), to see if I could think of a shorter string to explain the current error codes:
> 
> ```
> WS001     NO_ALLOC_SPACE    All devices fully allocated
>                 "Every device belonging to the pool has been fully allocated. "
>                 "To increase the allocable space, add additional data devices "
>                 "to the pool."
> 
> 
> EM001     NO_IPC_REQUESTS   Pool state changes not possible
>                 "The pool will return an error on any IPC request that could "
>                 "cause a change in the pool state, for example, a request to "
>                 "rename a filesystem. It will still be able to respond to "
>                 "purely informational requests."
> 
> 
> EM002     NO_POOL_CHANGES   Pool maintenance operations not possible
>                 "The pool is unable to manage itself by reacting to events, "
>                 "such as devicemapper events, that might require it to take "
>                 "any maintenance operations."
> ```

It would be easy to add an additional short description along with the full description.

**Comment by drckeefe** [comment]
I thought the truncated version was good, but I see in the version above there is a short description, which I  might be
better.   Here is an example.

$ stratis pool list --uuid=33123a6c-eb66-4c6f-9097-6b9521da4551
UUID: 33123a6c-eb66-4c6f-9097-6b9521da4551
Name: spool1
Alerts:
    WS001    All devices fully allocated
    EM001    Pool state changes not possible
    EM002    Pool maintenance operations not possible
Actions Allowed: fully_operational
Cache: No

Or it could be better to put all the alerts at the bottom of the output, especially if we want the full text version. 

$ stratis pool list --uuid=33123a6c-eb66-4c6f-9097-6b9521da4551
UUID: 33123a6c-eb66-4c6f-9097-6b9521da4551
Name: spool1
Actions Allowed: fully_operational
Cache: No
Alerts:
WS001     NO_ALLOC_SPACE    All devices fully allocated
                "Every device belonging to the pool has been fully allocated. "
                "To increase the allocable space, add additional data devices "
                "to the pool."

EM001     NO_IPC_REQUESTS   Pool state changes not possible
                "The pool will return an error on any IPC request that could "
                "cause a change in the pool state, for example, a request to "
                "rename a filesystem. It will still be able to respond to "
                "purely informational requests."

EM002     NO_POOL_CHANGES   Pool maintenance operations not possible
                "The pool is unable to manage itself by reacting to events, "
                "such as devicemapper events, that might require it to take "
                "any maintenance operations."

File: src/stratis_cli/_actions/_pool.py [hunk 1]
```diff
@@ -619,6 +619,53 @@ def my_func(value):
...
+        """
+        Alert summary to display, if any
+        :param mopool: object to access pool properties
+
+        :returns: string with alert summary
+        :rtype: str
+        """
+        error_codes = _List.alert_codes(mopool)
+
+        return (PoolErrorCode.summarize(code) for code in error_codes)
```
Comment (mulkieran) [inline src/stratis_cli/_actions/_pool.py:647]: Try this line instead:

```
return os.linesep.join(code.summarize() for code in error_codes)
```

**Review by mulkieran — CHANGES_REQUESTED** [review]

File: src/stratis_cli/_error_codes.py [hunk 2]
```diff
@@ -138,3 +159,12 @@ def explain(error_code):
         :param str error_code:
         """
         return PoolErrorCode.error_from_str(error_code).explain()
+
```
[the remark this answers is not in the document]
Comment (bgurney-rh) [reply src/stratis_cli/_error_codes.py:162]: I think this is still being used by the `stratis pool explain <error_code>` command.

File: src/stratis_cli/_actions/_pool.py (hunk 1 above)
Comment (bgurney-rh) [reply src/stratis_cli/_actions/_pool.py]: This seems to work; I now see this:
```
Alerts: All devices fully allocated
```

**Review by mulkieran — APPROVED** [review]

**Comment by bgurney-rh** [comment]
Squashed into a single commit.

**Pull request merged**